Makefile

```diff
@@ -13,6 +13,10 @@ ensure:
 lint:
 	./scripts/lint.sh
 
+.PHONY: format
+format:
```

Comment:
If you want an even shorter name, `fmt` has become a familiar abbreviation to many developers because of the `go fmt` command. For example, TF uses `terraform fmt`.

Reply:
id prefer to not abbreviate format, since we have lots of folks who will be contributing here, that are not developers